### ISSUE TITLE
The Alpha158 handler get NAN in y["label"]["LABEL0"] in other instruments except csi300

### ISSUE THREAD

**Issue by lzd-1230** [microsoft/qlib#1860]
## ❓ Questions and Help

After changing market from default "csi300" to other specific code like market = ["SH600519"] in example tutorial `workflow_by_code.ipynb`

I got 0 loss in model.fit(dataset)
![Image](https://github.com/user-attachments/assets/3eb2f57b-423f-448b-9b18-814999b64b6d)

After debug, I find that the output of y["LABEL0"] in  

https://github.com/microsoft/qlib/blob/f2caf452e95348ea0ab986dbdc88f2dd96751835/qlib/contrib/model/gbdt.py#L40C1-L41C1

are all NAN 

![Image](https://github.com/user-attachments/assets/a2ec2684-d4e3-4c99-bc58-cc3bd9ee189f)

and using default ”csi300“ is work fine with

![Image](https://github.com/user-attachments/assets/d65ac713-e173-4e5f-a9c7-67ddb122ccc1)

So how do i do model fit in such specific stock id，rather than sci300？

I'd appreciate it if someone can tell me Why is that happen and how to solve it?
